pkg/arvo/app/metadata-store.hoon

```diff
@@ -171,6 +176,14 @@
   :-  [group-path [app-name app-path]]
   (~(got by associations) [group-path [app-name app-path]])
 ::
+++  metadata-for-group
+  |=  =group-path
+  %-  ~(gas by *(map [^group-path resource] metadata))
+  %+  turn  ~(tap in (~(got by group-indices) group-path))
+  |=  =resource
+  :-  [group-path resource]
```

Comment:
Why do we include the `group-path` in the result here? In every place you get this result, you already know the `group-path`, and it's constant, so including it in the map key seems unnecessary.

Reply:
We're returning an "associations" type that's specifically scoped to the group.

Reply:
Ohh, I see. You probably want `~(gas by *assocations)` then, for clarity.

Reply:
That doesn't work, so I'm going to keep it as is.

Reply:
Figured it out!